hometask8/index.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
```

Comment:
Consider checking the following article for more convenient npm packages:
https://blog.cloudboost.io/adding-swagger-to-existing-node-js-project-92a6624b855b
Basically, swagger is usually added as an Express middleware using `app.use('/swagger', swaggerUi.serve, swaggerUi.setup('./link-to-your-swagger-file.json(yml)'));`